dateparser/timezones.py

```diff
@@ -6,7 +6,7 @@
 timezone_info_list = [
     {
         'regex_patterns':
-            [r'(\W|\d|_)\(?%s\)?$'],
+            [r'(\W|\d|_)\(?%s(\)|$|[^\w+])'],
```

Comment:
Sure, that gives us expression `(\W|\d|_)\(?%s($|\W)`. The revised expression only checks for either the end of string or not any word character next to timezone abbreviation. It might look repetitive to have another regex just to bifurcate for this binary difference. What do you think?